newUserBooking.cs

```diff
@@ -48,13 +54,11 @@ private void txtExitDate_ValueChanged(object sender, EventArgs e)
         {
        
         }
-
-        private void btnLogin_Click(object sender, EventArgs e)
+        private void btnSignIn_Click(object sender, EventArgs e)
         {
-            
-             
+            MapperCustomerDto();
```

Comment:
Burada veritabanının default verdiği müşterinin id'si ile ilgili hiç bir bilgi vermiyorum. Bu yüzden kayıt  işlemi  başarıyla olursa Customers form(isimini kesinlikle değiştirmeliyim🙃 )'una  yönlendirip id öğrenmeyi sağlayabiliriz diye düşünüyorum. sizce nasıl olur veya başka bir kurgu mu düşünmeliyim bunun için ?

Reply:
Buradaki isimlendirmelerini gozden gecir bence. Sign In diyorsun fakat kullanici bilgisi guncellemek icin kullaniyorsun. Yani kimse sign in olmuyor burada. Bir de kullaniciya customer id vermeyi neden istiyorsun?

Reply:
burada kullanıcı güncellemiyorum aslında.
MapperCustomer da saveCustomer'ı çağırdığım ve dto nesnesini verdiğim metot bu. müşteerinin  kaydını gerçekleştiriyorum. 

Room kaydı yaparken customer id'i vermem gerekiyor ya. eğer bilinirse girebilir id' yi gibi düşünmüştüm. fakat fazla manuel kaldı sanırım.

Reply:
kullanıcı olarak adlandırdığınız müşteri mi ?